packages/analytics-client-common/src/default-tracking.ts

```diff
@@ -57,25 +57,10 @@ export const isSessionTrackingEnabled = (defaultTracking: DefaultTrackingOptions
   return false;
 };
 
-/**
- * Returns page view tracking config
- *
- * if config.attribution.trackPageViews and config.defaultTracking.pageViews are both TRUE
- * then always track page views
- *
- * if config.attribution.trackPageViews is TRUE and config.defaultTracking.pageViews is FALSE
- * then only track page views on attribution
- *
- * if config.attribution.trackPageViews is FALSE and config.defaultTracking.pageViews is TRUE
- * then always track page views
- *
- * if config.attribution.trackPageViews and config.defaultTracking.pageViews are both FALSE
- * then never track page views
- */
 export const getPageViewTrackingConfig = (config: BrowserOptions): PageTrackingOptions => {
-  let trackOn: PageTrackingTrackOn | undefined = config.attribution?.trackPageViews ? 'attribution' : () => false;
+  let trackOn: PageTrackingTrackOn | undefined = () => false;
   let trackHistoryChanges: PageTrackingHistoryChanges | undefined = undefined;
-  let eventType: string | undefined = 'Page View';
+  let eventType: string | undefined;
 
   const isDefaultPageViewTrackingEnabled = isPageViewTrackingEnabled(config.defaultTracking);
```

Comment:
Does this still require
```
{
    defaultTracking: true
}
```
for pageview + attribution tracking to be enabled? Can we make these opt-out instead of opt-in? Basically make the out-of-the-box behavior the "correct" one for standard marketing web traffic use cases.

Reply:
correct it is still opt in. i haven't made these changes yet here but i plan to do so in another PR.